include/sound/pcm.h

```diff
@@ -110,25 +110,26 @@ struct snd_pcm_ops {
 #define SNDRV_PCM_RATE_11025		(1U<<2)		/* 11025Hz */
 #define SNDRV_PCM_RATE_16000		(1U<<3)		/* 16000Hz */
 #define SNDRV_PCM_RATE_22050		(1U<<4)		/* 22050Hz */
-#define SNDRV_PCM_RATE_32000		(1U<<5)		/* 32000Hz */
-#define SNDRV_PCM_RATE_44100		(1U<<6)		/* 44100Hz */
-#define SNDRV_PCM_RATE_48000		(1U<<7)		/* 48000Hz */
-#define SNDRV_PCM_RATE_64000		(1U<<8)		/* 64000Hz */
-#define SNDRV_PCM_RATE_88200		(1U<<9)		/* 88200Hz */
-#define SNDRV_PCM_RATE_96000		(1U<<10)	/* 96000Hz */
-#define SNDRV_PCM_RATE_176400		(1U<<11)	/* 176400Hz */
-#define SNDRV_PCM_RATE_192000		(1U<<12)	/* 192000Hz */
-#define SNDRV_PCM_RATE_352800		(1U<<13)	/* 352800Hz */
-#define SNDRV_PCM_RATE_384000		(1U<<14)	/* 384000Hz */
-#define SNDRV_PCM_RATE_705600		(1U<<15)	/* 705600Hz */
-#define SNDRV_PCM_RATE_768000		(1U<<16)	/* 768000Hz */
+#define SNDRV_PCM_RATE_24000		(1U<<5)		/* 24000Hz */
```

Comment:
You cannot do this.
Bit 7 was 48KHz before this patch, now it is 44.1KHz.
This will have catastrophic consequences with anything which uses audio.

Reply:
It should be possible to do this actually, SNDRV_PCM_RATE_ is kernel internal define